tests/Analyzers/AnalyzerAssemblyGenerator.cs

```diff
@@ -30,9 +37,19 @@ private static async Task<IEnumerable<MetadataReference>> GetReferencesAsync()
                 MetadataReference.CreateFromFile(typeof(CodeFixProvider).Assembly.Location),
             };
```

Comment:
📝 All of these should be removed and resolved from NuGet packages instead of from assemblies on disk.

Reply:
If you don't mind, let's do that in a follow-up as this is already existent in main. Would you mind filing an issue?